src/ML-POD/compute_pod_local.h

```diff
@@ -0,0 +1,60 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef COMPUTE_CLASS
+// clang-format off
+ComputeStyle(pod/ldd,ComputePODLocal);
```

Comment:
Either the command should be called compute pod/local or the class name should be changed to ComputePODLDD and the files to compute_pod_ldd.h/cpp
If the latter is done, also documentation and possibly examples need to be updated.

Reply:
The command is changed to compute pod/local. I updated the documentation and example to be consistent with the new command name.